fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -67,45 +67,29 @@ private val ICON_SIZE = 24.dp
 /**
  * Firefox Translations bottom sheet dialog.
  *
+ * @param translationsDialogState The current state of the Translations bottom sheet dialog.
+ * @param translatedPageTitle Title of the bottom sheet dialogue if the page was translated.
  * @param learnMoreUrl The learn more link for translations website.
- * @param showFirstTimeTranslation Whether translations first flow should be shown.
- * @param translateFromLanguages Translation menu items to be shown in the translate from dropdown.
- * @param translateToLanguages Translation menu items are to be shown in the translate to dropdown.
- * @param initialFrom The initial selection for the translate from dropdown.
- * @param initialTo The initial selection for the translate to dropdown.
- * @param translationError The type of translation errors that can occur.
  * @param onSettingClicked Invoked when the user clicks on the settings button.
  * @param onLearnMoreClicked Invoked when the user clicks on the "Learn More" button.
- * @param onTranslateButtonClicked Invoked when the user clicks on the "Translate" button.
- * @param onNotNowButtonClicked Invoked when the user clicks on the "Not Now" button.
+ * @param onPositiveButtonClicked Invoked when the user clicks on the positive button.
+ * @param onNegativeButtonClicked Invoked when the user clicks on the negative button.
  * @param onFromSelected Invoked when the user selects an item on the from dropdown.
  * @param onToSelected Invoked when the user selects an item on the to dropdown.
  */
-@Composable
 @Suppress("LongParameterList")
+@Composable
 fun TranslationsDialogBottomSheet(
+    translationsDialogState: TranslationsDialogState,
```

Comment:
Typically, I think we try to have Composables avoid using `State` parameters directly. I'm not sure I'm up on the latest of that discussion, @MozillaNoah may be able to comment more.

If we do keep this, can we compute the `translatedPageTitle` directly from the state?

Reply:
Done . I added translatedPageTitle in TranslationsDialogState.